include/net/nrf_cloud_coap.h

```diff
@@ -135,6 +135,18 @@ int nrf_cloud_coap_pgps_url_get(struct nrf_cloud_rest_pgps_request const *const
  */
 int nrf_cloud_coap_sensor_send(const char *app_id, double value, int64_t ts_ms);
 
+/**
+ * @brief Send raw bytes to nRF Cloud.
+ * @param[in]     app_id The app_id identifying the type of data.
+ * @param[in]     buf buffer with binary string.
+ * @param[in]     buf_len  length of buf in bytes.
+ * @param[in]     ts_ms  Timestamp the data was measured, or NRF_CLOUD_NO_TIMESTAMP.
```

Comment:
*question:* why enforce having a timestamp here? What if raw data is a collection of messages?

Reply:
I was unsure if this might be needed for compatibility so I pulled it in. Can be removed if unnecessary.